Sources/Sentry/SentryANRTrackingIntegrationV2.m

```diff
@@ -76,21 +76,20 @@ - (void)dealloc
     [self uninstall];
 }
 
-- (void)anrDetected
+- (void)anrDetected:(SentryANRType)type
```

Comment:
l: This is the type of thing I'm not a fan of in these minimalist PRs.
I'm not sure if this is a mistake because this argument is not being used, or if you plan to use it in another PR.

Reply:
I would say that has more to do with me not fully cleaning up this PR rather than small PRs 😅